.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+.DS_Store
+build
```

Comment:
This is just a minor change to ignore some files from the repo that are typically ignored.